tests/test_page.py

```diff
@@ -94,39 +100,102 @@ def test_wait_for_page_empty_base_url(selenium):
     assert isinstance(Page(selenium).wait_for_page_to_load(), Page)
 
 
-def test_find_element(page, selenium):
+def test_find_element_selenium(page, selenium, driver_interface):
+    skip_not_selenium(driver_interface)
```

Comment:
Rather than use `skip_not` calls, we could have a `selenium` fixture for tests that are specific to Selenium, `driver` fixture for tests that work with both, and `splinter` fixture for tests that are specific to Splinter. I think this improves readability and test collection. I would also suggest putting these driver specific tests into sub-directories named after the implementation.